docker-compose.yaml

```diff
@@ -1,4 +1,4 @@
-version: "3.9"  # optional since v1.27.0
+version: "3.3"  # optional since v1.27.0
```

Comment:
Is this version change here important? Or from something else?

Reply:
I was debating leaving this in or not. It's a one-line change that lets this run on older docker-compose (which I have)
I can open a separate PR for it.

Reply:
no need, just curious why the change was here :) fine to toss it in here as well since it doesn't seem to do any harm

Reply:
There were some other things I wanted to change in the test.sh script actually, so I just threw this change in there instead. Fits better.